ansible/roles/enomoloch-get-pcaps/files/router_key

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCsjJo07J3jaWEXkc1ORj6i5cwuR2E8B7A8rT8kd8t5mm+mibgHSXVN5VXOVbQMU7Kzierzc+C9uGqkuTH2iNTeT+ZLPvXhjDGpw2uiBGP3qtzRN2a6E4QnELcHwX2h3IJOSQSpUUknKgS+zyAjYvNsWBALw8CCEvaHRdYylAs/l8VNI11VTq6um15uZYwOYN/uWsqwpF8idRHdHy1820eChY4QpXaXgfGmKGUnuTA2oORRIc84/+F5tvte1XdW/BJpSyFHiJFQM93lua0KhXu7FzpvXMF73W9hzufLvBY+4b9fvUo1mZvIxdqjVsLevyTzwfLBt8FU6qJxxT+yMokB2vq8gXi+ivjuebbfOaGQQ6uyg/aVWE2TqinOKKC9Xsm5OutQVmqIkGTleAtp5bfBhxCUF/xmYtCIDKiXDNrEfkkZg5ZHEj9nQuwS8CDAqirkU/pccBjb2kQBc6+S/Bw0qk/6iI495sAfBINaz9U4McbJhncU4bHU841mFu7jn20= jannitschke@Jans-MBP
```

Comment:
you do realize this repository is public?

Reply:
We can just do this locally in a config file or potentially even through terraform provisioning: https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/private_key